<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>火车车次查询系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>信息输入</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QLineEdit" name="Tra_Num_Line">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>车次：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CheckBtn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>471</width>
      <height>181</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>查询结果</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="DetailBtn">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>93</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>详细信息</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>401</width>
       <height>111</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>tid</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ttype</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>始发站</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>终点站</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>481</width>
      <height>121</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>所有列车信息</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>37</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>火车类型：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>263</x>
       <y>38</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>火车号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Ttype_LineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Tid_LineEdit">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="FirstBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>｜&lt;第一条</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PrevBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;&lt;前一条</string>
     </property>
    </widget>
    <widget class="QPushButton" name="NextBtn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>下一条&gt;&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LastBtn">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>80</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>最后一条&gt;|</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>552</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
